### ISSUE TITLE
Flutter web Sentry stacktrace does not show full code

### ISSUE THREAD

**Issue by ahmed-ashraf** [getsentry/sentry-dart#1657]
### Description

I use this command below to build the app for web
flutter build web --dart-define=environment=dev --web-renderer canvaskit --release --source-maps

Then I use this command to upload source maps
flutter packages pub run sentry_dart_plugin

Here is the flutter version from flutter doctor
Flutter (Channel stable, 3.7.10, on Microsoft Windows [Version 10.0.22621.525], locale en-US)
    • Flutter version 3.7.10 on channel stable at D:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4b12645012 (6 months ago), 2023-04-03 17:46:48 -0700
    • Engine revision ec975089ac
    • Dart version 2.19.6
    • DevTools version 2.20.1

I got those errors on sentry log. The hidden url in the image is the url of the website. It seems that it needs to point to the code.

![image](https://github.com/getsentry/sentry-dart/assets/19362719/c6c12905-0607-4bb3-ade6-4f7a19de2bae)

The stacktrack doesn't show the full code but it views the line of error like the image below. How can I fix that?
![image](https://github.com/getsentry/sentry-dart/assets/19362719/195bf5b7-4d95-480f-9f13-f6ac3b988c82)

**Comment by stefanosiano** [comment]
@krystofwoldrich can you have a look into it?

**Comment by krystofwoldrich** [comment]
@ahmed-ashraf I've tried the same commands with our Flutter Web Example and the application code is symbolicated.

https://sentry-sdks.sentry.io/issues/4527428170/events/f53a147339884003ad8ca51962e641d0/

Could you share what version of the Dart SDK and the Dart Plugin are you using and what is the output of the source maps upload command?

Or if possible could you share a minimal reproducible example, that would help us investigate the issue faster?

**Comment by stefanosiano** [comment]
Hi @ahmed-ashraf 
Could you share what version of the Dart SDK and the Dart Plugin are you using and what is the output of the source maps upload command?
The best would be if you could share a reproducible example

**Comment by ahmed-ashraf** [comment]
Hi @stefanosiano 
This is the Dart SDK
Dart SDK version: 2.19.6 (stable) (Tue Mar 28 13:41:04 2023 +0000) on "windows_x64"
Here is the dart plugin version on android studio 222.4560

### Here is the source map upload command output

`Downloading sentry-cli from https://downloads.sentry-cdn.com/sentry-cli/2.20.6/sentry-cli-Windows-x86_64.exe to .dart_tool/pub/bin/sentry_dart_plugin\sentry-cli.exe

Downloaded Sentry CLI binary checksum verification passed successfully (hash: HASH_VALUE).

Sentry CLI downloaded successfully.
☑ reading config values                                                             
☑ validating config values                                                             
[❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚] 18% uploading debug symbols..                                
> Found 1 debug information file

> Resolved source code for 0 debug information files

> Prepared debug information file for upload

> Nothing to upload, all files are on the server
☑ uploading debug symbols                                                             

Created release 1.11.0@1.11.0+1110
[❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚] 27% uploading source maps..                               
> Found 9 files

> Analyzing 9 sources

> Rewriting sources

> Adding source map references

> Bundled 9 files for upload
> Bundle ID: e7013275-b144-5cec-8e66-b0c839bba905

> Uploaded files to Sentry

> File upload complete (processing pending on server)
> Organization: focalagent
> Project: propertybox
> Release: 1.11.0@1.11.0+1110
> Dist: 1110
> Upload type: artifact bundle



Source Map Upload Report
  Scripts
    ~/app.js
    ~/assets/packages/flutter_dropzone_web/assets/flutter_dropzone.js
    ~/flutter.js
    ~/flutter_service_worker.js
  Minified Scripts
    ~/assets/packages/flutter_image_compress_web/assets/pica.min.js (sourcemap at ../../../../main.dart.js.map)
    ~/canvaskit/canvaskit.js (sourcemap at ../main.dart.js.map)
    ~/canvaskit/profiling/canvaskit.js (sourcemap at ../../main.dart.js.map)
    ~/main.dart.js (sourcemap at main.dart.js.map)
  Source Maps
    ~/main.dart.js.map

> Found 341 files

> Analyzing 341 sources

> Rewriting sources

> Adding source map references

> Bundled 341 files for upload
> Bundle ID: 849bd5c4-d91b-5244-a78f-293cf12255b9

> Uploaded files to Sentry

> File upload complete (processing pending on server)
> Organization: focalagent
> Project: propertybox
> Release: 1.11.0@1.11.0+1110
> Dist: 1110
> Upload type: artifact bundle

Source Map Upload Report
  Scripts
    ~/aad_oauth-0.1.9/lib/aad_oauth.dart
	...

~/aad_oauth-0.1.9/lib/request/token_request.dart
    ...

~/lib/config/colors.dart
    ...
    

~/lib/modules/buttons.dart
    ...
    

~/lib/modules/notification/settings.dart
    ~/lib/modules/...

~/lib/modules/dialog_payg.dart

~/lib/modules/.../widgets/do_and_do_not_notice.dart
    ~/lib/modules/edit_bar.dart
	...
        
  Minified Scripts

~/lib/modules/.../plan_tab.dart (no sourcemap ref)

- warning: could not determine a source map reference (Could not auto-detect referenced sourcemap for ~/lib/modules/...plan_tab.dart)
    ~/lib/utils/extensions/iterable_extension.dart (no sourcemap ref)
      - warning: could not determine a source map reference (Could not auto-detect referenced sourcemap for ~/lib/utils/extensions/iterable_extension.dart)
☑ uploading source maps

Could not determine any commits to be associated with a repo-based integration. Proceeding to find commits from local git tree.

Success! Set commits for release 1.11.0@1.11.0+1110

Finalized release 1.11.0@1.11.0+1110`

**Comment by stefanosiano** [comment]
Thanks for sending them!

**Comment by buenaflor** [comment]
Hi @ahmed-ashraf I have trouble reproducing this, do you have a small/minimal reproducible project example we could take a look at on GitHub? that would be the most helpful, thanks!

**Comment by buenaflor** [comment]
Closing this in favour of #1679
